charts/alertmanager/Chart.yaml

```
@@ -6,7 +6,7 @@ icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/a
 sources:
   - https://github.com/prometheus/alertmanager
 type: application
-version: 1.5.0
+version: 1.5.1
```

adding a feature should raise minor version

```suggestion
version: 1.6.0
```